shell/app/node_main.cc

```diff
@@ -165,7 +184,15 @@ int NodeMain(int argc, char* argv[]) {
       versions.SetReadOnly(ELECTRON_PROJECT_NAME, ELECTRON_VERSION_STRING);
     }
 
-    node::LoadEnvironment(env);
+    // TODO(codebytere): we should try to handle this upstream.
+    {
+      node::InternalCallbackScope callback_scope(
+          env, v8::Local<v8::Object>(), {1, 0},
```

Comment:
How do we know what the correct field values are for this async_context?

Reply:
This should match what is being run in [`node_main_instance`](https://github.com/nodejs/node/blob/master/src/node_main_instance.cc#L126-L130); i confirmed this with Anna Henningsen :)